### PR TITLE
Updates include improved project docs, regions, app keys, and regions

### PR DESCRIPTION
**Pull request by leggetter**
- Homepage has been updated to highlight Database and Search as products and their underlying features. Gives us more room to highlight features.
- Updated project docs
- Reviewed the use of Tigris URL throughout the docs
- Updated Docusaurus to latest

### CODE REVIEW

File: docs/guides/nextjs/add-existing-app/index.mdx [hunk 1]
```diff
@@ -352,18 +354,17 @@ export default async function handler(
...
 
-:::tip Built-in search
-Tigris makes it really easy to implement [search](/docs/sdkstools/typescript/database/search)
-within your applications by providing a serverless full-text search engine that
-makes all your data instantly searchable.
-:::
+:::tip Built-in search Tigris makes it really easy to implement
+[search](/docs/sdkstools/typescript/database/search) within your applications by
+providing a serverless full-text search engine that makes all your data
+instantly searchable. :::
```
Comment (leggetter) [inline docs/guides/nextjs/add-existing-app/index.mdx:360]:
```suggestion
:::tip
Built-in search Tigris makes it really easy to implement
[search](/docs/sdkstools/typescript/database/search) within your applications by
providing a serverless full-text search engine that makes all your data
instantly searchable.
:::
```

File: docs/sdkstools/cli/authentication.md [hunk 2]
```diff
@@ -2,28 +2,28 @@
...
 ## User authentication
 
-Every request on the cloud Tigris platform must be authenticated.
-To authenticate yourself using CLI, run the following command:
+Every request on the cloud Tigris platform must be authenticated. To
+authenticate yourself using CLI, run the following command:
 
 ```shell
-tigris login api.preview.tigrisdata.cloud
+tigris login api.preview.tigrisdata.cloud # TODO: is there a central login?
```
Comment (leggetter) [inline docs/sdkstools/cli/authentication.md:9]: @efirs is there a central login or do you point to a region?

File: docs/sdkstools/cli/authentication.md [hunk 3]
```diff
@@ -53,21 +54,20 @@ token: <token value>
 url: api.preview.tigrisdata.cloud
 ```
 
-:::caution
-Please make sure to keep the token secure. Whoever possess the token
-can execute any command on your name.
-:::
+:::caution Please make sure to keep the token secure. Whoever possess the token
+can execute any command on your name. :::
```
Comment (leggetter) [inline docs/sdkstools/cli/authentication.md:58]:
```suggestion
:::caution 
Please make sure to keep the token secure. Whoever possesses the token
can execute any command on your name. 
:::
```

File: docs/sdkstools/typescript/database/branching.mdx [hunk 4]
```diff
@@ -77,29 +81,31 @@ const db = tigrisClient.getDatabase();
...
 ```
 
-If you only want to delete some data or drop an entire collection, use [collection operations](delete.mdx).
+If you only want to delete some data or drop an entire collection, use
+[collection operations](delete.mdx).
 
-:::note
-The `main` database branch can never be deleted.
-:::
+:::note The `main` database branch can never be deleted. :::
```
Comment (leggetter) [inline docs/sdkstools/typescript/database/branching.mdx:100]:
```suggestion
:::note

The `main` database branch can never be deleted.

:::
```

File: docs/guides/nextjs/add-existing-app/index.mdx (hunk 1 above)
Comment (leggetter) [inline docs/guides/nextjs/add-existing-app/index.mdx:360]:
```suggestion
:::tip

Built-in search Tigris makes it really easy to implement
[search](/docs/sdkstools/typescript/database/search) within your applications by
providing a serverless full-text search engine that makes all your data
instantly searchable.

:::
```

File: docs/sdkstools/cli/authentication.md [hunk 5]
```diff
@@ -87,6 +87,5 @@ authentication configuration:
 }
 ```
 
-:::caution
-Please make sure to keep the client id and secret in a safe place.
+:::caution Please make sure to keep the client id and secret in a safe place.
 :::
```
Comment (leggetter) [inline docs/sdkstools/cli/authentication.md:91]:
```suggestion
:::caution

Please make sure to keep the client id and secret in a safe place.

:::
```

File: docs/sdkstools/cli/authentication.md (hunk 2 above)
Comment (efirs) [reply docs/sdkstools/cli/authentication.md:9]: You point to a region.

api....I can't remember....tigrisdata.cloud

We can extract it to the TIGRIS_REGION variable later as well.

Comment (leggetter) [inline docs/sdkstools/cli/authentication.md:9]:
```suggestion
tigris login api.preview.tigrisdata.cloud
```

**Pull request merged**